teuthology/provision/cloud/base.py

```diff
@@ -13,7 +13,7 @@
 
 
 class Provider(object):
-    _driver_posargs = list()
+    _driver_posargs = list() # type: ignore
```

Comment:
How does it work, why is this required, I am actually not familiar with 'mypy'.

Reply:
See answer below.